lib/parallel_tests/cli.rb

```diff
@@ -88,6 +88,19 @@ def report_results(test_results)
       results = @runner.find_results(test_results.map { |result| result[:stdout] }*"")
       puts ""
       puts @runner.summarize_results(results)
+      report_failure_rerun_commmand(test_results)
+    end
+
+    def report_failure_rerun_commmand(test_results)
+      failing_sets = test_results.select{|test_result| test_result[:exit_status] != 0 }
+      return unless failing_sets.size > 0
+
+      puts "\n\nTests have failed for a parallel_test group. Use the following command to run the group again:\n\n"
+      failing_sets.each do |failing_set|
+        command = failing_set[:command]
+        command = command + " --seed #{failing_set[:seed]}" if failing_set[:seed]
```

Comment:
maybe `@runner.seed_command(set[:seed])` then each runner can have it's custom logic ?